packages/access-control/src/lib/strategies/google/controllers/google-strategy.controller.ts

```diff
@@ -12,11 +12,16 @@ export function buildGoogleController({
 }: IGoogleAccountBuildController): IController {
   @Controller()
   class GoogleStrategyController {
+    @UseGuards(AuthGuard('google'))
     @Get(routeGoogle)
+    async googleAuth() {
```

Comment:
unnecesary end-point.

Reply:
I was testing Google login functioning without this endpoint and the behavior wasn't stable, so I recommend keep it for the normal Google oauth2 strategy flow